questions/truth-table.md

```diff
@@ -0,0 +1,94 @@
+### What is a truth table? Try to recreate the following Boolean expression using different operators in JavaScript.
+
+```js
+a && b
+```
+
+#### Answer
+
+A truth table is a mathematical table used in logic when working with Boolean expressions. 
+
+Intuitively, we logically construct a truth table when working with logical operations.
```

Comment:
The `Intuitively, we logically...` confuses me a bit... I'm curious if it can't be better written as `We need to logically...`